app/jobs/card_action_required_email_job.rb

```diff
@@ -0,0 +1,20 @@
+class CardActionRequiredEmailJob
+  include Sidekiq::Worker
+  attr_accessor :subscription
+
+  def perform(subscription_id)
+    @subscription = Subscription.find(subscription_id)
+    return if subscription.blank?
```

Comment:
This is redundant, find raises if the record isn't there

Reply:
Just line 6 is redundant correct?

Reply:
just line 7 - subscription will never be blank if you use find

```suggestion
```